lib/xing/nominal/database_config_validator.rb

```diff
@@ -0,0 +1,31 @@
+require 'xing/nominal/yaml_config_validator'
```

Comment:
IMO services should be utils for the larger Rails app. nominal and snapshot are both support which should be extracted to own gems - separating their files will make that easier.